src/scripts/get_lkg_hash.py

```diff
@@ -52,6 +54,10 @@ def get_latest_hash():
 
     return latest_git_hash
 
+retry_strategy = Retry(
```

Comment:
Might want to move this up to line 21 so it's defined before the function body of get_recent_builds.